`make setup` failed on macOS

**Describe the bug**

<!-- In what way is Lighthouse behaving incorrectly or unexpectedly? -->

The command `make setup` failed on macOS.
When I ran the command, the following error message appeared.

<details><summary>Click to expand to show the error message.</summary>

```
$ make setup 
id: illegal option -- -
usage: id [user]
       id -A
       id -F [user]
       id -G [-n] [user]

       id -P [user]
       id -g [-nr] [user]
       id -p [user]
       id -u [-nr] [user]
id: illegal option -- -
usage: id [user]
       id -A
       id -F [user]
       id -G [-n] [user]

       id -P [user]
       id -g [-nr] [user]
       id -p [user]
       id -u [-nr] [user]
docker-compose build --pull --build-arg USER_ID= --build-arg GROUP_ID=
```

</details>

`id` command on macOS doesn't have `--user` and  `--group` options, so the environment variables `USER_ID` and `GROUP_ID` was set empty.

`docker-compose build` was executed, but `docker-compose up --detach` failed because of the following error.

<details><summary>Click to expand to show the message that said the setup failed.</summary>

```
docker-compose up --detach
[+] Running 21/21
 ✔ redis 8 layers [⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                        9.2s 
   ✔ c57ee5000d61 Already exists                                           0.0s 
   ✔ 01e3a03de6cc Pull complete                                            1.4s 
   ✔ 1fb20eb53a94 Pull complete                                            0.6s 
   ✔ 7e3dfb4dc4b5 Pull complete                                            2.2s 
   ✔ 7edd0a34dadd Pull complete                                            3.9s 
   ✔ 0a0eb0ff91b9 Pull complete                                            2.3s 
   ✔ 4f4fb700ef54 Pull complete                                            3.7s 
   ✔ b56c0201b4fa Pull complete                                            3.8s 
 ✔ mysql 11 layers [⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                   54.2s 
   ✔ 20e4dcae4c69 Pull complete                                           24.0s 
   ✔ 1c56c3d4ce74 Pull complete                                            4.2s 
   ✔ e9f03a1c24ce Pull complete                                            5.3s 
   ✔ 68c3898c2015 Pull complete                                            6.8s 
   ✔ 6b95a940e7b6 Pull complete                                            6.8s 
   ✔ 90986bb8de6e Pull complete                                            7.7s 
   ✔ ae71319cb779 Pull complete                                           23.7s 
   ✔ ffc89e9dfd88 Pull complete                                            8.8s 
   ✔ 43d05e938198 Pull complete                                           19.8s 
   ✔ 064b2d298fba Pull complete                                           20.8s 
   ✔ df9a4d85569b Pull complete                                           21.5s 
[+] Running 5/5
 ✔ Network lighthouse_default    Created                                   0.1s 
 ✔ Container lighthouse-node-1   Started                                   1.2s 
 ✔ Container lighthouse-mysql-1  Started                                   1.2s 
 ✔ Container lighthouse-redis-1  Started                                   1.2s 
 ✔ Container lighthouse-php-1    Created                                   0.1s 
Error response from daemon: unable to find user so16: no matching entries in passwd file
make: *** [up] Error 1
```

</details>

I checked this error on following environment.

- OS: macOS 13.6.4 (22G513)
- processor: 2.3 GHz Dual-Core Intel Core i5
- docker-compose: v2.23.3-desktop.2
- make: GNU Make 3.81

**Expected behavior/Solution**

`id` command should finish successfully and the error message should NOT appear when `make setup` executed.
And then, `docker-compose up --detach` should finish successfully.

**Steps to reproduce**

1. Execute `make setup` on macOS

**Lighthouse Version**

master (a97436fbf375f8ba90d865b236ed05617083c212)

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ setup: build docs/node_modules vendor ## Setup the local environment
 
 .PHONY: build
 build: ## Build the local Docker containers
-	docker-compose build --pull --build-arg USER_ID=$(shell id --user) --build-arg GROUP_ID=$(shell id --group)
+	docker-compose build --pull --build-arg USER_ID=$(shell id -u) --build-arg GROUP_ID=$(shell id -g)
 
 .PHONY: up
 up: ## Bring up the docker-compose stack

